nlp_apps.ipynb

```diff
@@ -450,14 +448,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 36,
    "metadata": {
     "collapsed": true
    },
    "outputs": [],
    "source": [
-    "wordseq = words(federalist)\n",
-    "wordseq = wordseq[114:-3098]"
+    "wordseqs = words(federalist)\n",
+    "wordseqs = wordseqs[114:-3098]"
```

Comment:
Was it necessary to change the name of the variable?

Reply:
Wasn't `wordseq` already present in the repository?
Anyway, its fine if it makes things simpler.